src/db/database.ts

```
@@ -0,0 +1,58 @@
+            return rows;
+        } catch (error) {
+            throw error;
+        }
+    }
+
+    public async ExecutaComandoNoQuery(sql: string, parametros: any[] = []): Promise<any> {
+        try {
+            const [result] = await this.pool.promise().query(sql, parametros);
+            return result;
+        } catch (error) {
+            throw error;
```

Biome (1.9.4)

[error] The catch clause that only rethrows the original error is useless.

An unnecessary catch clause can be confusing.
Unsafe fix: Remove the try/catch clause.


(lint/complexity/noUselessCatch)